lib/node/runner/checker.go

```diff
@@ -329,6 +329,7 @@ func FinishedBallotStore(c common.Checker, args ...interface{}) (err error) {
 			checker.Ballot,
 			checker.NodeRunner.Consensus().TransactionPool,
 			checker.Log,
+			checker.NodeRunner.Log(),
```

Comment:
I remember, `BallotChecker.Log` is from `NodeRunner.Log`, right?

Reply:
Yes but it has more informations.
```
	checker.Log = checker.Log.New(logging.Ctx{
		"ballot":   checker.Ballot.GetHash(),
		"state":    checker.Ballot.State(),
		"proposer": checker.Ballot.Proposer(),
		"round":    checker.Ballot.Round(),
		"from":     checker.Ballot.Source(),
		"vote":     checker.Ballot.Vote(),
	})
```
I think, these informations are not fit for `NewBlock`. So I used NodeRunner.Log().